OptimizelySDK/Bucketing/DecisionService.cs

```diff
@@ -43,6 +43,22 @@ public class DecisionService
         private IErrorHandler ErrorHandler;
         private UserProfileService UserProfileService;
         private ILogger Logger;
+        private UserProfile _userProfile;
+
+        private bool _decisionBatchInProgress = false;
+
+        public bool DecisionBatchInProgress
+        {
+            get => _decisionBatchInProgress;
+            set
+            {
+                _decisionBatchInProgress = value;
```

Comment:
Quick note for future Mike: You probably need to check that we're going from `true` to `false`.